app/Config/App.php

```diff
@@ -279,14 +280,14 @@ class App extends BaseConfig
 
 	/**
 	 * --------------------------------------------------------------------------
-	 * Cookie HTTP Only
+	 * Cookie HttpOnly
 	 * --------------------------------------------------------------------------
 	 *
 	 * Cookie will only be accessible via HTTP(S) (no JavaScript).
 	 *
 	 * @var boolean
 	 */
-	public $cookieHTTPOnly = false;
+	public $cookieHTTPOnly = true;
```

Comment:
I think that the default property value should be the same as the setcookie() function, equal to "false"

Reply:
I set it to true for better security. I think the framework should lean on secure practices for apps.